SU-HSDO/suhumsci
# Shs 108 name overlap mobile

## Summary
Overrides some inherited styles from an external 'decanter' component. The original styles would allow the 'School of...' text to overflow onto the menu button.

## Steps to Test
1. This is a good candidate for Tugboat testing as it doesn't need any new pages. 
2. Go to the homepage from the link below and go down to mobile size in browser and make sure the text doesn't overflow onto the 'Menu' button.
3. Do this for both colorful/traditional

## Screenshots
![Screen Shot 2022-09-27 at 1 53 04 PM](https://user-images.githubusercontent.com/8405274/192601294-c293d9aa-210a-4184-847d-7b6e06d8e0be.png)

## Notes
The original and real issue with this is that menu button is outside of document flow. It is `absolute` and then pulled up with a negative `top`. This is less than ideal and leaves an opening for more potential wonkiness in the future. 

I spent some time refactoring the menu nav button to be in proper DOM order, however the changes were getting extensive, touched many files, and the JS functionality for controlling the open/close state of the menu started to behave inconsistently. Unfortunately, to do this the proper way I think we need to factor the entire V1 mega menu. I preserved the work I was doing in a branch, but I think the time vs. benefit here ended up going beyond the ticket scope.

Leaving base branch as `develop` until new sprint branch.
